index.html

```diff
@@ -1032,20 +1037,20 @@ <h2>Issuer</h2>
       </section>
 
       <section>
-        <h2>Proofs (e.g., Signatures)</h2>
+        <h2>Proofs (Signatures)</h2>
```

Comment:
The Verification "Proofs" section has the "e.g.". They should probably do the same thing one way or the other.

Reply:
Yes, @davidlehn. My intended upcoming review of Section 6 Advanced Topics will suggest the same change.